src/routes/dashboard/contracts/+page.svelte

```diff
@@ -0,0 +1,91 @@
+<script lang="ts">
+    import { contractStore } from '$lib/stores/contractStore'
+    import { generateContractClient } from '$lib/utils/contractUtils'
+    import { SorobanRpc } from '@stellar/stellar-sdk'
+
+    let contractId = ''
+    let contractName = ''
+    let error = ''
+    let loading = false
+
+    const server = new SorobanRpc.Server('https://soroban-testnet.stellar.org')
+
+    async function handleLoadContract() {
+        loading = true
+        error = ''
+
+        try {
+            const contract = await generateContractClient(contractId, server)
+            $contractStore.setCurrentContract(contract)
```

Comment:
Since we're using a function from the `contractStore` object, the `$` in front causes an error on this line, since it's looking for a _value_ from the store.
```suggestion
            contractStore.setCurrentContract(contract)
```